config/dynamic.c

```diff
@@ -295,6 +293,8 @@ const struct luaL_reg dylib_lua_c_funcs [] = {
 	{"list_gk_fib6",         l_list_gk_fib6},
 	{"list_gk_neighbors4",   l_list_gk_neighbors4},
 	{"list_gk_neighbors6",   l_list_gk_neighbors6},
+	{"list_arp_cache",       l_list_arp_cache},
+	{"list_nd_cache",        l_list_nd_cache},
```

Comment:
The names `list_lls_arp()` and `list_lls_nd()` are better because GT blocks too have ARP and ND caches and we'll likely have functions to list those caches too down the line.